slsim/Deflectors/DeflectorTypes/nfw_hernquist.py

```diff
@@ -26,23 +26,29 @@ def velocity_dispersion(self, cosmo=None):
         :type cosmo: ~astropy.cosmology class
         :return: velocity dispersion [km/s]
         """
+        if "vel_disp" in self._deflector_dict.columns:
```

Comment:
this line caused an error in the CI. Here you require the _deflector_dict object to be an astropy table, while in the input the format is a dictionary (compatible with astropy table, but not vise versa)